### PR TITLE
chore(releasing): Prepare v0.37.1 release

### PR DESCRIPTION
**Pull request by jszwedko**
Checklist: https://github.com/vectordotdev/vector/issues/20258

Only one actual fix, the rest are build pipeline fixes picked over from `master`.

### CODE REVIEW

**Review by tobz — APPROVED** [review]
LGTM.

File: website/cue/reference/releases/0.37.0.cue [hunk 1]
```diff
@@ -33,7 +33,12 @@ releases: "0.37.0": {
 
 	known_issues: [
 		"""
-			The `parse_ddtags` setting added to the `datadog_agent` Source incorrectly parses the tags into an object instead of an array.
+			The `geoip2` enrichment table type stopped handling `GeoLite2-City` mmdb types. This is fixed in v0.37.1.
+			""",
+		"""
+			The `parse_ddtags` setting added to the `datadog_agent` source incorrectly parses the
+			tags into an object instead of an array. The `datadog_logs` sink also failed to
```
Comment (buraizu) [inline website/cue/reference/releases/0.37.0.cue:40]:
```suggestion
			tags into an object instead of an array. The `datadog_logs` sink also fails to
```

**Review by buraizu — APPROVED** [review]
Thanks, approving with one minor update requested

**Comment by jszwedko** [comment]
Merged locally to preserve commit history.

**Pull request closed without merging**